### PR TITLE
Docs: Add license

### PR DESCRIPTION
**Pull request by dijonkitchen**

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,674 @@
...
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+Also add information on how to contact you by electronic and paper mail.
+
+  If the program does terminal interaction, make it output a short
+notice like this when it starts in an interactive mode:
+
+    <program>  Copyright (C) <year>  <name of author>
```
Comment (sdwolfz) [inline LICENSE.md]: This should be replaced with the appropriate names.

**Comment by dijonkitchen** [comment]
Can this be merged now

**Comment by sdwolfz** [comment]
Ping @syl20bnr as all legal matters should be approved by the repository owner.

**Comment by eklitzke** [comment]
Shouldn't this file be named `LICENSE`? It's not a markdown file.

**Pull request closed without merging**